src/main/resources/openapi/model/api.yaml

```diff
@@ -13,3 +13,53 @@ tags:
 #    description: "local host"
 
 paths:
+
+
+  #get email by reference ID
+  /notifications/email/{reference}:
+    get:
+      description: Get an email by reference ID
+      operationId: getEmailByReference
+      parameters:
+        - name: reference
+          in: path
+          description: email reference ID
+          required: true
+          schema:
+            type: integer
+            format: int64
+      tags:
+        - notificationService
```

Comment:
can we separate and make emailNotificationService separate from smsNotificationService